XP-Pen_Artist16_Pro/dumps/hid_report_descriptors.txt

```diff
@@ -0,0 +1,37 @@
+006:002:002:DESCRIPTOR         1526447794.376711
+ 06 0A FF 09 01 A1 01 85 02 09 02 75 08 95 09 15
+ 00 26 FF 00 81 02 09 03 75 08 95 09 15 00 26 FF
+ 00 91 02 C0 06 0A FF 09 01 A1 01 85 04 09 02 75
+ 08 95 07 15 00 26 FF 00 81 02 09 03 75 08 95 07
+ 15 00 26 FF 00 91 02 C0
+
+006:002:001:DESCRIPTOR         1526447794.381658
+ 05 0D 09 02 A1 01 85 07 09 20 A1 00 09 42 09 44
+ 09 45 15 00 25 01 75 01 95 03 81 02 95 02 81 03
+ 09 32 95 01 81 02 95 02 81 03 75 10 95 01 35 00
+ A4 05 01 09 30 46 EF 34 27 73 86 00 00 81 42 09
+ 31 46 EE 1D 26 05 4C 81 42 B4 09 30 45 00 26 FF
+ 1F 81 42 09 3D 15 81 25 7F 75 08 95 01 81 02 09
+ 3E 15 81 25 7F 81 02 C0 C0 05 0D 09 01 A1 01 85
+ 08 09 20 A1 00 09 42 09 44 09 45 15 00 25 01 75
+ 01 95 03 81 02 95 02 81 03 09 32 95 01 81 02 95
+ 02 81 03 75 10 95 01 35 00 A4 05 01 09 30 46 EF
+ 34 27 73 86 00 00 81 42 09 31 46 EE 1D 26 05 4C
+ 81 42 B4 09 30 45 00 26 FF 1F 81 42 09 3D 15 81
+ 25 7F 75 08 95 01 81 02 09 3E 15 81 25 7F 81 02
+ C0 C0
+
+006:002:000:DESCRIPTOR         1526447794.387651
+ 05 01 09 02 A1 01 85 09 09 01 A1 00 05 09 19 01
+ 29 03 15 00 25 01 95 03 75 01 81 02 95 05 81 01
+ 05 01 09 30 09 31 26 FF 7F 95 02 75 10 81 02 05
+ 0D 09 30 26 FF 1F 95 01 75 10 81 02 C0 C0 05 01
+ 09 02 A1 01 09 01 A1 00 85 01 05 09 19 01 29 05
+ 95 05 75 01 15 00 25 01 81 02 95 03 81 01 05 01
+ 09 30 09 31 95 02 75 10 16 00 80 26 FF 7F 81 06
+ 09 38 15 81 25 7F 95 01 75 08 81 06 05 0C 0A 38
+ 02 95 01 75 08 81 06 C0 C0 05 01 09 06 A1 01 85
+ 06 05 07 19 E0 29 E7 15 00 25 01 75 01 95 08 81
+ 02 05 07 19 00 29 FF 26 FF 00 75 08 95 06 81 00
+ C0
```

Comment:
Please split report descriptors into separate files, one per interface. For this tablet and all the others in this PR. This way it's easier to decode or process a specific report descriptor with command-line tools.

Reply:
Yeah, I put one comment because I am going to add the `probe` while fixing this at the same time